client/static/js/hatohol_server_edit_dialog_parameterized.js

```diff
@@ -192,6 +194,13 @@ HatoholServerEditDialogParameterized.prototype.createMainElement = function() {
     mainDiv.append('<form id="add-server-param-form" class="form-horizontal" role="form" autocomplete="off">');
     return mainDiv;
   }
+
+  function prioritySort(serverTypes) {
```

Comment:
We use the naming convention in which a method name begins with verb.
According to it, the name becomes sortByPriority().